vipps-login-api-faq.md

```diff
@@ -271,13 +273,44 @@ Vipps Login from phone number are reserved for such special cases and needs to b
+
+| Item                                                 | Description                                                                                                                                                                    | Example                                                                           | Comments                                                                                               |
+|------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| Top text                                             | Specifies the purpose of collecting the consents                                                                                                                               | "Merchant AS would like to send you tailored offers in several digital channels." |                                                                                                        |
+| Subset of consents you would like to ask for         | We offer a set of predefined consents. See [When using delegatedConsents, which consents does Vipps support?](#when-using-delegatedconsents-which-consents-does-vipps-support) | email, sms, digital, personal                                                     | If you want a consent type that we currently don't support, reach out to us at accessuserinfo@vipps.no |
+| Links to your membership terms and privacy statement | We need to check your terms to make sure the flow is not used to mislead or abuse the end user                                                                                 | www.merchant.com/termsandconditions <br /> www.merchant.com/privacystatement      |                                                                                                        |
+| Recipient to sign DPA                                | Vipps will function as a data processor and not have any ownership to the data involved. For more information, please visit merchant terms and conditions                      | recipient@merchant.com                                                            |                                                                                                        |
+
+## When using delegatedConsents, which consents does Vipps support?
+
+We have built support for several consents, and the merchant may select a subset of these to include in their own flow as well as select which of the consents that are required/optional.
+If we do not currently support the consents you look for, you may make a request to accessuserinfo@vipps.no and we'll be in touch.
```

Comment:
I would replace bare email addresses and html links with links everywhere in the markdowns.